python/ray/data/context.py

```diff
@@ -125,6 +125,9 @@
 # If disabled, users can still manually print stats with Dataset.stats().
 DEFAULT_AUTO_LOG_STATS = False
 
+# Whether stats logs should be verbose.
```

Comment:
```suggestion
# Whether stats logs should be verbose. This will include fields such
# as `extra_metrics` in the stats output, which are excluded by default.
```